configs/parent.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://github.com/apache/trafficserver/tree/master/configs/parent.schema.json",
+  "title": "Traffic Server parents configuration",
+  "description": "parent configuration file structure. Licensed under Apache V2 https://www.apache.org/licenses/LICENSE-2.0",
+  "definitions": {
+    "destination": {
+      "type": "string",
+      "properties": {
+        "anyOf": [
+          { "dest_domain": { "type": "string" } },
+          { "dest_host": { "type": "string" } },
+          { "dest_ip": { "type": "string"} },
+          { "url_regex": { "type": "string"} }
+        ]
+      }
+    },
+    "internal": {
+      "type": "string",
+      "properties": {
+        "anyOf": [ "true", "false" ]
+      }
+    },
+    "max_simple_retries": {
+      "type": "number",
+      "properties": {
+        "anyOf": [ 1, 2, 3, 4, 5 ]
+      }
+    },
+    "max_unavailable_server_retries": {
+      "type": "number",
+      "properties": {
+        "anyOf": [ 1, 2, 3, 4, 5 ]
+      }
+    },
+    "method": {
+      "type": "string",
+      "properties": {
+        "anyOf": [ "get", "post", "put", "trace" ]
+      }
+    },
+    "parent_retry": {
+      "type": "string",
+      "properties": {
+        "anyOf": [ "simple_retry", "unavailable_retry", "both" ]
+      }
+    },
+    "qstring": { 
+      "type": "string",
+      "properties": {
+        "anyOf": [ "consider", "ignore" ]
+      }
+    },
+    "round_robin": {
+      "type": "string",
+      "properties": {
+        "anyOf": [ "true", "false", "strict", "consistent_hash", "latched" ]
+      }
+    },
+    "scheme": { 
+      "type": "string",
+      "properties": {
+        "anyOf": [ "http", "https" ]
+      }
+    },
+    "secondary_mode": {
+      "type": "number",
+      "properties": {
+        "anyOf": [ 1, 2, 3 ]
+      }
+    },
+    "parent_record": {
+      "type": "object",
+      "properties": {
+        "required": { "$ref": "#/definitions/destination" },
+        "anyOf": [
```

Comment:
This means a `parent_record` can have only one of the values in `anyOf`. Is that intended?

Reply:
I fixed the parent.schema.json file.  It now validates correctly